src/Perlang.Tests.Integration/Typing/StringTests.cs

```diff
@@ -42,16 +38,9 @@ public void string_variable_can_be_reassigned()
             .Be("this is another string");
     }
 
-    [SkippableFact]
+    [Fact]
     public void ascii_string_inferred_variable_can_be_reassigned_with_non_ascii_value()
     {
-        // The code below is incredibly hard to support in compiled mode, because: an AsciiString cannot be assigned to a
-        // String variable in C++ (because the latter is an abstract class; I believe the C++ compiler will try to make a
-        // copy of it). `const perlang::string& s = ...` works, but then the problem is that the variable can obviously
-        // not be reassigned on the second line... because it is constant. We'll have to think through how to solve this
-        // properly.
-        Skip.If(PerlangMode.ExperimentalCompilation, "Not yet supported in compiled mode");
-
```

Comment:
:point_up: This is the only test that didn't work in this test class. The other ones were just a matter of removing the skipping; they would already pass with our current code base.